src/thunderbird_accounts/authentication/middleware.py

```diff
@@ -65,7 +64,7 @@ def __call__(self, request: HttpRequest):
 
         end = time.perf_counter_ns()
 
-        logging.debug(f'> total time {(end - start) / 1000000} ms')
+        sentry_sdk.set_measurement('cached_allowed_host_get_time', (end - start) / 1000000, 'millisecond')
```

Comment:
Got annoyed at some logging, switched it to sentry metrics. We probably don't need this anymore but easy to ship it to sentry instead.